### ISSUE TITLE
Clicking on Version v2.4.0 leads to an undefined commit on https://telescope.cdot.systems/

### ISSUE THREAD

**Issue by raygervais** [Seneca-CDOT/telescope#2650]
**What happened**:

When you select the version, it's redirects to an undefined (404) commit

![image](https://user-images.githubusercontent.com/14265337/148476120-05ca0431-1edd-4a10-b649-02f1bda5a522.png)

**What should have happened**:

When you select the version, it's supposed to take you to the GitHub commit relating to that release version

**Comment by manekenpix** [comment]
Some context:

We process the payload sent by GitHub [here](https://github.com/Seneca-CDOT/telescope/blob/cfc8f8f30f67869715f3dceed836cd76c4c76275/tools/autodeployment/webhook.js#L66), so we can  pass the sha to our deployment script [here](https://github.com/Seneca-CDOT/telescope/blob/cfc8f8f30f67869715f3dceed836cd76c4c76275/tools/autodeployment/builds.js#L75), and this sha is added to the `env` file (deployment, staging, production) used in the deployment.

The payload sent when a push event takes place includes an `after` prop that contains the commit sha used in this process (see [here](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push)). This prop doesn't exist in the payload sent for release events (see [here](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#release)).

We need to find a way to get the commit sha from a payload sent during a release event.

**Comment by manekenpix** [comment]
Checking the payloads sent by github, it seems their documentation might not be up to date.
The docs about release payloads say that there's a prop called `target_commitish` that's supposed to look like this:

```json
"tag_name": "0.0.1",
"target_commitish": "master",
"name": null,
```
but checking our logs, we get this:

```json
"tag_name": "2.5.0",
"target_commitish": "321f0afe7b78bf7e0fb774719b1cb3a231ca5d8b",
"name": "Release 2.5.0",
```
I think it'd be a good idea to try `target_commitish` as a way to get the release's commit sha.

**Comment by manekenpix** [comment]
Closed via #2756
